jina/proto/jina.proto

```diff
@@ -285,6 +285,16 @@ message DocumentArrayProto {
     repeated DocumentProto docs = 1; // a list of Documents
 }
 
+/**
+ * Represents an entity (like a Pea)
+ */
+message RelatedEntity {
+    string id = 1; // unique id of the entity, like the name of a pea
+    string address = 2; // address of the entity, could be an IP address
```

Comment:
both port and address seem redundant?

Reply:
address should be IP, domain etc - maybe I should it make more clear in the comment. This does not include the port